tests/mineral_test.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+from pycoal import mineral
+
+def test_normalize():
+    pass
```

Comment:
as [mentioned](https://github.com/capstone-coal/pycoal/pull/23#discussion_r100511791), normalize is a function from `ndarray P` to `ndarray Q` such that `P.shape == (M, N, B)` and `Q.shape == (M, N, num_layers)`. additionally, the least and greatest elements should be within the wavelength range of a given test image. also, the distribution should be the principal components of the input matrix. i can see at least two tests for this:
* to test normalized spectra, make sure that the input is MxNxB and the output is MxNx`num_layers` using the `shape` property
* to test constrained wavelength, verify for some `wavelengthList` of the test input that
    * `Q[0,0,0]` equals `wavelengthList[indexOfGreaterThan(wavelength_min)]`, and that
    * `Q[0,0,num_layers-1]` equals `wavelengthList[indexOfGreaterThan(wavelength_max)]`

you could also test that it works for both 1x1xB pixels and MxNxB images. it would also be possible to test that the resulting values are indeed the principal components of the image, but i think the dimensions and bounds are most important.